build/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     go build -o /go/bin ./cmd/connaisseur
 
 FROM alpine:3 as certs
-RUN apk --update --no-cache add ca-certificates=20240226-r0
+RUN apk --update --no-cache add ca-certificates
```

Comment:
If you think the rule in general is reasonable, maybe
```suggestion
# hadolint ignore=DL3018
RUN apk --update --no-cache add ca-certificates
```
instead of ignoring it everywhere